client/src/App.js

```diff
@@ -1,5 +1,21 @@
-function App() {
-  return <h1>Serientracker</h1>
+import styled from 'styled-components'
```

Comment:
Just my personal preference:
```suggestion
import styled from 'styled-components/macro'
```